src/sync/installation.ts

```diff
@@ -259,7 +260,7 @@ async function doProcessInstallation(app, data: BackfillMessagePayload, sentry:
 		for (const perPage of [20, 10, 5, 1]) {
 			// try for decreasing page sizes in case GitHub returns errors that should be retryable with smaller requests
 			try {
-				return await processor(logger, github, gitHubInstallationClient, jiraHost, repository, cursor, perPage);
+				return await processor(logger, github, gitHubInstallationClient, jiraHost, repository, cursor, perPage, data);
```

Comment:
This was the least evil way I could think of..... passing around the message payload, because of the cute way that this code exists all task processors being shared etc this was the most generic solution I found, basically just  goodies bag for everything now. 😂

Reply:
yeah, this is an acceptable solution.  This whole sync functionality needs to be redone...